source/common/router/router.cc

```diff
@@ -1481,6 +1512,11 @@ void Filter::UpstreamRequest::onResetStream(Http::StreamResetReason reason,
                                             absl::string_view transport_failure_reason) {
   ScopeTrackerScopeState scope(&parent_.callbacks_->scope(), parent_.callbacks_->dispatcher());
 
+  if (span_ != nullptr) {
+    // Add tags about reset.
+    span_->setTag(Tracing::Tags::get().Error, Http::Utility::resetReasonToString(reason));
```

Comment:
The `error` tag is a boolean, so would be better to have a separate tag to define the reason.

Reply:
Same: Added a new tag